server/shellquote/shellquote.go

```diff
@@ -0,0 +1,156 @@
+// Based on https://github.com/kballard/go-shellquote
```

Comment:
Why not use the package? (out of curiosity) Is it because it's not active?

Reply:
With all the supply chain attacks, the current trend is to reduce the amount of dependencies for security. The 1 function I needed was pretty small to bring in.